PDSA_System/Client/Pages/TeamlederPage.razor

```diff
@@ -0,0 +1,180 @@
+                            <button type="button" class="close" @onclick="@ModalCancel">&times;</button>
+                        </div>
+
+                        <!-- Modal body -->
+                        <div class="modal-body">
+                            <p>This is the modal content!</p>
+                            <div>
+                                <label for="formGroupExampleInput">AnsattNr</label>
+                                <input type="number" class="form-control" @bind="_teamMedlemskap.AnsattNr" id="formGroupExampleInput" placeholder="AnsattNr"/>
+                            </div>
+                            <div>
+                                <label for="formGroupExampleInput">TeamId</label>
+                                <input type="number" class="form-control" @bind="_teamMedlemskap.TeamId" id="formGroupExampleInput" placeholder="TeamId"/>
+                            </div>
+                        </div>
+
+                        <!-- Modal footer -->
+                        <div class="modal-footer">
+                            <button type="button" class="btn" @onclick="AddBrukerToTeam">Legg til i team</button>
+                            <button type="button" class="btn" @onclick="@ModalCancel">Avbryt</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        }
+
+        @if (showModalDelete)
+        {
+            @*Delete Modal *@
+        <div class="modal fade show" id="myModal" style="display:block" aria-modal="true" role="dialog">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+
+                        <!-- Modal Header -->
+                        <div class="modal-header">
+                            <h4 class="modal-title">Bekreft</h4>
+                            <button type="button" class="close" @onclick="@ModalCancelDelete">&times;</button>
+                        </div>
+                    <!-- Modal body -->
+                    <div class="modal-body">
+                            <div></div>
+                               <div class="form-group">
+                             <h6>Bekreft at du ønsker å fjerne denne brukeren</h6>
+                             <p>Dette kan ikke omgjøres</p>
+                             <div>
+                            </div>
+                    </div>
+                        </div>
+                         <!-- Modal footer -->
+                        <div class="modal-footer">
+                            <button type="button" class="btn" @onclick="@ModalCancelDelete">Avbryt</button>
+                            <button type="button" class="btn btn-danger" @onclick="@(()=>DeleteItem(_teamMedlemskap.AnsattNr ,_teamMedlemskap.TeamId))">Slett</button>
+                    </div>
+                        </div>
+                        </div>
+                        </div>
+
+         }
+    	@*</Authorized>
```

Comment:
Om det funker, så bør det gjerne implementeres igjen, bare for å holde uautensierte folk unna